back/app/models/initiative.rb

```diff
@@ -68,6 +71,9 @@ class Initiative < ApplicationRecord
     post.after_validation :set_assigned_at, if: ->(record) { record.assignee_id && record.assignee_id_changed? }
   end
 
+  # def should
```

Comment:
We probably want to remove this?

Reply:
Why do we need to change this? Don't think it has anything to do with the author

Reply:
Duh, sorry. Was looking at the wrong bit of code.